plugin/storage/es/spanstore/reader.go

```diff
@@ -60,8 +63,60 @@ func NewSpanReader(client es.Client, logger *zap.Logger) *SpanReader {
 
 // GetTrace takes a traceID and returns a Trace associated with that traceID
 func (s *SpanReader) GetTrace(traceID model.TraceID) (*model.Trace, error) {
-	// TODO
-	return nil, nil
+	return s.readTrace(traceID.String(), spanstore.TraceQueryParameters{})
```

Comment:
i did this to follow cassandra syntax + i didn't want to call `GetTrace` in `FindTraces`